workspace/api/openfdd_bridge/rule_kit.py

```
@@ -420,6 +421,140 @@ def build_rule_kit_zip(
+        zf.writestr(
+            "README.md",
+            "# Open-FDD export all rules\n\n"
+            "Per-rule folders under `rules/<rule_id>/` mirror the single-rule Rule Lab kit.\n"
+            "See `manifest.json` for per-rule status and warnings.\n",
+        )
+        try:
+            from .fault_catalog import catalog_payload
+
+            zf.writestr("catalog/fault_codes.json", json.dumps(catalog_payload(), indent=2))
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)